content/en/istio-ca-certs-integrations/acmpca-integration/_index.md

```diff
@@ -14,44 +14,81 @@ The [`getistio gen-ca`](/getistio-cli/reference/getistio_gen-ca) command furnish
+    1. Click **Advanced** to expand the options.
+    1. Select **RSA 2048**.
+    1. Click **Next**.
+1. On the "Configure certificate revocation" step, click **Next**.
+1. On the "Add tags" step, click **Next**.
+1. On the "Configure CA permissions" step, click **Next**.
+1. On the "Review" step, select the confirmation check box, and click **Confirm and create** button to create ACM Private CA.
+1. Click the **Create** button to create the CAS.
+
+Before ACM Private CA can start issuing certificates, you need to activate by installing a CA certificate. 
+
+1. From the "Private CAs" page, click the **Install a CA certificate to active your CA** link.
+1. Change the validity to **365 days**.
+1. Select **SHA256WITHRSA** from the Signature algorithm list.
+1. Click **Next**.
+1. Click the **Confirm and install** button to generate, and install the root CA certificate.
 
-We can supply parameters related to connecting to ACM Private CA and CSR creation through a YAML config file or command-line options. Creating a config file is recommended.
+The figure below shows the Private CA page.  Note that yours might look different if you configured your own CA subject name.
 
-Here's an example of a YAML config file with explained parameters.
+![Private CA Page](./acm-private-ca.png)
 
+### Configure AWS credentials
+
+Ensure you have AWS credentials set up with the `AWSCertificateManagerPrivateCAFullAccess` and `AWSCertificateManagerFullAccess` policy attached on a machine you're accessing the Kubernetes cluster from. Alternatively, if you installed GetIstio on AWS Cloud Shell, the credentials are already set up.
+
+</details>
+
+## Creating ACM configuration
+
+We will use a YAML configuration to configure ACM Private CA. Use the YAML below as a template, and enter the ACM Private CA information from the AWS console:
+ 
 ```yaml
 providerName: "aws"
 providerConfig:
   aws:
-    signingCAArn: {your acmpca CA ARN}
-   # template to use for the CSR.
+    # This will hold the ARN value from the Details page of ACM Private CA 
+    signingCAArn: "  arn:aws:acm-pca:us-east-2:859085711342:certificate-authority/097162cc-6a9e-47ab-b5e0-fecf32556d6d"
     templateArn: "arn:aws:acm-pca:::template/SubordinateCACertificate_PathLen0/V1"
-   # Optional field. If left blank would default to SHA256WITHRSA
-   signingAlgorithm: SHA256WITHRSA
-
+    signingAlgorithm: SHA256WITHRSA
 certificateParameters:
   secretOptions:
-    # Namespace where 'cacerts' Kubernetes secret is created on your target cluster
-    istioCANamespace: "istio-system"
-    # SecretFilePath is the file path used to store the Kubernetes Secret in yaml format
-    secretFilePath:
-    # Force flag when enabled forcefully deletes the `cacerts` secret
-    # in istioNamespace, and creates a new one.
-    force: true
+    istioCANamespace: "istio-system" # namespace where `cacerts` secrets live
+    force: true # force delete the `cacerts` secret and replace it with this new one
```

Comment:
Just a note - need to change this flag if the other PR in getistio is released to bintray.

Reply:
👍🏼